.github/workflows/gitlab-runners-ubi8-git-publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: ubi8-git-publish
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - gitlab-runners/ubi8-git/version.json
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+    # Only build on redhat-cop repo, do not attempt to build on forks
+    if: github.repository == 'redhat-cop/containers-quickstarts'
```

Comment:
why not make the repo a variable and get rid of this IF that way people can test building images with the action on their forks by setting up the destination repo variable?

Reply:
I think if we remove the IF, this workflow will trigger every time you push to master on your own fork (to sync with upsteam) and there's been changes to `gitlab-runners/ubi8-git/version.json`. 

At the same time, I can see that you want to be able to test this push to <insert registry here> for testing or other purposes. I don't have a great solution to this but maybe a few options
- keep the IF and make the destination repo a variable
- make a copy of this workflow that you use in your fork (without the IF)

Did I get that right? Thoughts? @itewk @garethahealy

Reply:
@pabrahamsson by default when you fork a repo with github actions the actions are all disabled by default and you have to manually go to the actions tab on your fork and click through a couple dialogs to enable them. I think thats the perfet baclance if we add in a readme for which environment variables people need to specify on their fork to be able to have their fork build and push the images to their own quay account for testing and such.

Reply:
I like that approach @itewk, I'll see if I can update this PR with those suggestions later today.

Reply:
Updated